Adding setting default channel to yaml based install (#4263)
knative/docs: docs/admin/install/eventing/install-eventing-with-yaml.md

        <!-- TODO(https://github.com/knative/docs/issues/2153): Add more Channels here -->

> ### Setting a default channel

Reviewer: maybe say this is an example?

> ### Setting a default channel

Reviewer 2: ```suggestion
### Setting a default Channel
```

> If you are using different channel implementations, like the KafkaChannel, or you want a specific configuration of the InMemoryChannel to be the default configuration, you can change the default behavior by updating the `default-ch-webhook` ConfigMap.

Reviewer 2: ```suggestion
If you are using different channel implementations, such as KafkaChannel, or you want a specific configuration of the InMemoryChannel to be the default configuration, you can change the default behavior by updating the `default-ch-webhook` ConfigMap.
```

```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: default-ch-webhook
  namespace: knative-eventing
data:
  default-ch-config: |
  clusterDefault:
      apiVersion: messaging.knative.dev/v1beta1
      kind: KafkaChannel
      spec:
      numPartitions: 10
      replicationFactor: 1
  namespaceDefaults:
      my-namespace:
      apiVersion: messaging.knative.dev/v1
      kind: InMemoryChannel
      spec:
          delivery:
          backoffDelay: PT0.5S
          backoffPolicy: exponential
          retry: 5
```

> **NOTE:** The `clusterDefault` setting determines the global, cluster-wide default channel type. You can configure channel defaults for individual namespaces by using the `namespaceDefaults` setting.

Reviewer: IMO this should be moved to the previous section.

> **NOTE:** The `clusterDefault` setting determines the global, cluster-wide default channel type. You can configure channel defaults for individual namespaces by using the `namespaceDefaults` setting.

Reviewer 2: ```suggestion
!!! note
    The `clusterDefault` setting determines the global, cluster-wide default Channel type. You can configure Channel defaults for individual namespaces by using the `namespaceDefaults` setting.
```
Also needs mkdocs formatting

> ### Setting the default channel for the broker

Reviewer 2: ```suggestion
### Setting the default Channel for the Broker
```
These should always be caps

        <!-- TODO(https://github.com/knative/docs/issues/2153): Add more Channels here -->

### Setting a default Channel

If you are using different channel implementations, such as KafkaChannel, or you want a specific configuration of the InMemoryChannel to be the default configuration, you can change the default behavior by updating the `default-ch-webhook` ConfigMap.

```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: default-ch-webhook
  namespace: knative-eventing
data:
  default-ch-config: |
  clusterDefault:
      apiVersion: messaging.knative.dev/v1beta1
      kind: KafkaChannel
      spec:
      numPartitions: 10
      replicationFactor: 1
  namespaceDefaults:
      my-namespace:
      apiVersion: messaging.knative.dev/v1
      kind: InMemoryChannel
      spec:
          delivery:
          backoffDelay: PT0.5S
          backoffPolicy: exponential
          retry: 5

> ```

Reviewer 2: I'd add some comments or call outs here that explain what's being updated and what it does?

!!! note 

  The `clusterDefault` setting determines the global, cluster-wide default Channel type. You can configure Channel defaults for individual namespaces by using the `namespaceDefaults` setting.

### Setting the default Channel for the Broker

If you are using a Channel-based broker, you can change the default Channel type for the Broker from InMemoryChannel to KafkaChannel, by updating the `config-br-default-channel` ConfigMap.

```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: config-br-default-channel
  namespace: knative-eventing
spec:
  config:
    config-br-default-channel:
      channelTemplateSpec: |
        apiVersion: messaging.knative.dev/v1beta1
        kind: KafkaChannel
        spec:
          numPartitions: 6
          replicationFactor: 1
```

> 

Reviewer 2: Again, I'd make it clear what settings are actually modified to achieve this goal
